tests/caasp/oci_password.pm

```diff
@@ -18,7 +18,7 @@ use testapi;
 sub run {
     my ($self) = @_;
 
-    send_key 'alt-a';
+    check_var('VIDEOMODE', 'text') ? send_key 'alt-a' : send_key 'alt-w';
```

Comment:
This works for CaaSP 2.0 but this will break the Maintenance QA tests for CaaSP 1.0
See: http://ultron.suse.de/tests/632#step/oci_password/2

Can you think for a workaround maybe?

Reply:
This PR should do https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/3580. Please review.